src/main/java/de/orchound/gameoflife/GameOfLifeApplet.java

```diff
@@ -120,4 +122,14 @@ private void changeSpeed(long target) {
 	private float getInitialScale(int boardWidth, int boardHeight, int screenWidth, int screenHeight) {
 		return min((float) screenWidth / boardWidth, (float) screenHeight / boardHeight);
 	}
+
+	private void resetBoard() {
+		board.reset();
+		boardRenderer.update();
```

Comment:
You should reset the `gameTimeAccumulator` here as well, to make sure the new board state will last for a complete simulation update tick.
```suggestion
		boardRenderer.update();
		gameTimeAccumulator = 0;
```